.pipelines/ci.yml

```diff
@@ -37,19 +37,19 @@ stages:
       matrix:
         py311-1163:
           python.version: '3.11'
-          torch.version: 'torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu'
+          torch.version: 'torch torchvision torchaudio'
           ort.version: '1.16.3'
         py310-1151:
           python.version: '3.10'
-          torch.version: 'torch torchvision==0.14.1+cpu torchaudio==0.13.1 --index-url https://download.pytorch.org/whl/cpu'
+          torch.version: 'torch torchvision torchaudio'
           ort.version: '1.15.1'
         py39-1141:
           python.version: '3.9'
-          torch.version: 'torch==1.13.1+cpu torchvision==0.13.1+cpu torchaudio==0.12.1+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html'
+          torch.version: 'torch==1.13.1+cpu torchvision==0.14.1+cpu torchaudio==0.13.1 --index-url https://download.pytorch.org/whl/cpu'
           ort.version: '1.14.1'
         py38-1131:
           python.version: '3.8'
-          torch.version: 'torch==1.12.0+cpu torchvision==0.12.0+cpu torchaudio==0.11.0+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html'
+          torch.version: 'torch==1.12.1+cpu torchvision==0.13.1+cpu torchaudio==0.12.1+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html'
```

Comment:
should it be 1.13.1

Reply:
for torch version? no i tried that in this previous commit and it did not work: https://github.com/microsoft/onnxruntime-extensions/pull/617/commits/ea97836827bf5793334e0a331e23e4aebfd72adc - i believe vision version has to be one ahead of torch/audio version.

Reply:
the combination was come from pytorch installation page for the version prior to 2.x, not sure if it can be changed.

Reply:
i think it's still the same: https://pytorch.org/get-started/previous-versions/

Reply:
i believe the errors in the CI now is because ort is missing, previous errors were due to torch/torchvision versions - trying to see if they are fixed by adding ort back to requirements-dev.txt